test/Microsoft.Framework.FileSystemGlobbing.Tests/FunctionalTests.cs

```diff
@@ -29,6 +29,17 @@ public void Dispose()
         }
 
         [Fact]
+        public void DuplicatePatterns()
```

Comment:
Absolute path is not allowed. There are tests coverage ensuring the exceptions are thrown.

The path `\` can be very possible when a project is developed on Windows and move to *nix unless we abolish all the usage of `\`.